### ISSUE TITLE
Differrenciate lib to be copied and injected

### ISSUE THREAD

**Issue by ludohenin** [mgechev/angular2-seed#120]
Seperate both libs to be copied only and libs to be injected from the configuration.
I propose something like this:
```javascript
{
  lib_inject: [
    //...
  ],
  lib_copy_only: [
    //...
  ]
}
```

**Comment by mgechev** [comment]
I come up with this solution as well, I don't see anything smarter currently.

**Comment by ludohenin** [comment]
We could also do something like this
```
{
  lib: [ { path: 'node_modules/...', injectable: true } ]
}
```
but it adds extra complexity IMO

**Comment by jerryorta-dev** [comment]
When dividing grunt tasks, my intent was to experiment with different workflows that allow for environment flexibility, such as adding angular material, less, sass, etc. This is where I current am with a divergence ( not forked since out of scope ) of this project:
https://github.com/UIUXEngineering/angular2-hybrid-seed/tree/feature/workflow-2/web/tools

It's not fully fleshed out yet, but there may be something that may spark an idea for this issue.

**Comment by jerryorta-dev** [comment]
For the record, I think @ludohenin ideas are better for this project. Just putting something on the table to think over.

**Comment by mgechev** [comment]
@jerryorta-dev thanks for sharing!

@ludohenin yeah, lets go with the first idea! :-)

**Comment by ludohenin** [comment]
Yep. I'll implement while migrating gulp tasks to ts

**Comment by ludohenin** [comment]
This is addressed [here](https://github.com/mgechev/angular2-seed/commit/a61415a820eb87eeabbbdaa90c244d4d76be098c)

**Comment by jerryorta-dev** [comment]
That is looking sweet!

**Comment by ludohenin** [comment]
Solved by #128
